third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -232,7 +234,9 @@ impl TestConfig {
                 with_copy_inference: true,
             }));
             pipeline.add_processor(Box::new(ReferenceSafetyProcessor {}));
```

Comment:
We can revisit this later, but we probably could split critical edges early, and get rid of them as late as possible.

Reply:
Done. Actually, this is a bug since it will invalidate annotations from previous passes. It's not causing any problems because there are no critical edges in the test cases.